lib/ex_doc/formatter/html.ex

```diff
@@ -48,6 +48,8 @@ defmodule ExDoc.Formatter.HTML do
   defp generate_sidebar_items(modules, exceptions, protocols, output) do
     input = for node <- [%{id: "modules", value: modules}, %{id: "exceptions", value: exceptions}, %{id: "protocols", value: protocols}], !Enum.empty?(node.value), do: node
     content = Templates.sidebar_items_template(input)
+    # Safely reduce the size of the JS code
+    content = Regex.replace(~r/(\s|\n|\h|\v)/, content, "")
```

Comment:
@josevalim There is a gain on this hack ;D, (e.g. ~400% size reduction if we compare with the current `sidebar_items.js` in Elixir/master) and also this is to be consistent with the current `app.js` (also *uglify*). But I agree with you in be more conservative, can we add to the `:` and `,` list the following characters `[`, `]`, `{`, `}` and `;`?